.github/workflows/proto.yml

```
@@ -118,7 +118,7 @@ jobs:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)